tests/testthat/test-fotmob.R

```diff
@@ -59,7 +59,7 @@ test_that("fotmob_get_league_matches() works", {
     tidyr::unnest_wider(c(home, away), names_sep = "_")
 
   expect_gt(nrow(epl_ll_league_matches_unnested), 0)
-  expect_equal(ncol(epl_ll_league_matches_unnested), 7)
+  expect_equal(ncol(epl_ll_league_matches_unnested), 5)
```

Comment:
just tried running the test myself and i also only see 5 columns, so this seems good 👍 . i honestly can't remember what the other 2 columns were, but i'm not surprised to see that the output has changed, given how things have changed with fotmob before

```
# A tibble: 760 × 5
   match_id home_id home_name      away_id away_name   
   <chr>    <chr>   <chr>          <chr>   <chr>       
 1 3900932  9826    Crystal Palace 9825    Arsenal     
 2 3900933  9879    Fulham         8650    Liverpool   
 3 3900934  8678    Bournemouth    10252   Aston Villa 
 4 3900935  8463    Leeds          8602    Wolves      
 5 3900937  10261   Newcastle      10203   Nottm Forest
 6 3900938  8586    Tottenham      8466    Southampton 
 7 3900939  8668    Everton        8455    Chelsea     
 8 3900936  8197    Leicester      9937    Brentford   
 9 3900940  10260   Man United     10204   Brighton    
10 3900941  8654    West Ham       8456    Man City    
# … with 750 more rows
```